components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -1325,8 +1325,7 @@ document.observe('xwiki:dom:loaded', function()
             } else if (selected == 'yes') {
               var element = event.findElement('.entry');
               if (!element.down('.descendents') || element.hasClassName('collapsed')) {
```

Comment:
@sdumitriu As per my earlier comment about parents being expanded when we click 'Y' and producing a possibly confusing tree, this like could be changed to
`if ((!element.down('.descendents') || element.hasClassName('collapsed')) && element.down('.expand-tool')) {`